src/ucs/vfs/fuse/vfs_fuse.c

```diff
@@ -421,7 +416,7 @@ static void *ucs_vfs_fuse_thread_func(void *arg)
                          un_addr.sun_path, ucs_status_string(status));
             }
         } else {
-            ucs_warn("failed to connect to vfs socket '%s': %m",
+            ucs_diag("failed to connect to vfs socket '%s': %m",
```

Comment:
Why downgrade this to `diag`? As `ucs_vfs_sock_get_address` always succeeds and returns a valid address now, this `else` branch represents, at least, a real warning?

Reply:
since VFS is not critical for UCX usage, in case there is another problem with connecting to VFS socket no need to create false alarm..